app/modules/account/components/profile-form.jsx

```diff
@@ -0,0 +1,200 @@
+/**
```

Comment:
Who did rename this file? https://github.com/platformio/platformio-home/tree/develop/app/modules/account/components